### PR TITLE
Fix/refactor of randy comments

### PR DESCRIPTION
**Pull request by sstraatemans**
Fixes #
refactors some code after sitting with Randy

### CODE REVIEW

**Comment by Randynamic** [comment]
Randy look at the code.....Randy it takes too long... Randy here you go!!!...... "107 files changed" 😆 🤣

File: src/App.js [hunk 1]
```diff
@@ -14,61 +11,19 @@ import configureStore from './store';
...
-      👈
-    </span>
-    <span role="img" aria-label="8-ball">
-      🎱
-    </span>
-    <span role="img" aria-label="right hand">
-      👉
-    </span>
-  </Rotate>
+const App = () => (
```
Comment (Randynamic) [inline src/App.js]: So clean hehehe nice 
image

File: src/actions/competitors.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
+import { database } from './../store';
+import constants from './../components/shared/constants';
+
+//find the head2head score of the 2 competitors
+export const calcHead2Head = (competitorA, competitorB) => {
+  const streak = competitorA.results && competitorA.results[competitorB.uid];
+  let wins = 0;
+  let losses = 0;
+  if (streak) {
```
Comment (Randynamic) [inline src/actions/competitors.js]: Do you only get wins when you are on a streak? or is it a different kind a streak?

File: src/actions/competitors.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  );
+  if (presence) {
+    competitor.online = presence[competitor.uid];
+  }
+  return Object.assign({}, competitor);
+};
+
+const getRanking = (competitor, rankings, rankingIndex) => {
+  if (rankings) {
+    const setRankings = rankings && rankings[rankingIndex];
```
Comment (Randynamic) [inline src/actions/competitors.js]: is `setRankings` the current ranking?

File: src/actions/competitors.js (hunk 2 above)
Comment (sstraatemans) [reply src/actions/competitors.js]: This is about the head2head.
hm yeah..naming things. should just call it results

File: src/actions/competitors.js (hunk 3 above)
Comment (sstraatemans) [reply src/actions/competitors.js]: hm well currentRankings is also not a good name. its the ranking you are currently working with...if you know what I mean

File: src/components/CreateMatch/components/CompetitorAlert/index.js [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+      return prev;
+    });
+    return streak;
+  };
+
+  const setVictoryName = streak => {
+    if (streak >= 10) {
+      return 'godlike';
+    } else if (streak >= 6) {
+      return 'rampage';
```
Comment (Randynamic) [inline src/components/CreateMatch/components/CompetitorAlert/index.js]: image

File: src/components/CreateMatch/components/CompetitorAlert/index.js [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+    } else if (streak >= 3) {
+      return 'dominating';
+    }
+    return 'victory';
+  };
+
+  // let victoryName = this.setVictoryName(
```
Comment (Randynamic) [inline src/components/CreateMatch/components/CompetitorAlert/index.js]: Are you still going to use this

Comment (sstraatemans) [reply src/components/CreateMatch/components/CompetitorAlert/index.js]: yes

File: src/components/CreateMatch/components/CompetitorAlert/index.js (hunk 4 above)
Comment (sstraatemans) [reply src/components/CreateMatch/components/CompetitorAlert/index.js]: laugh all you want...its gonna be awesome

File: src/actions/competitors.js (hunk 3 above)
Comment (Randynamic) [reply src/actions/competitors.js]: nooo 🤣 but lets keep it like this hahahaha

**Review by Randynamic — APPROVED** [review]

File: src/actions/competitors.js (hunk 3 above)
Comment (sstraatemans) [reply src/actions/competitors.js]: hahaha

**Pull request merged**